taxonomies/unused/convert-wikidata-igp.py

```diff
@@ -0,0 +1,61 @@
+import json
+import re
+
+# Load the query.json file
+with open('/Users/pierre/development/openfoodfacts-server/taxonomies/food/query-off-ing.json', 'r') as f:
+    query_data = json.load(f)
+
+# Create a dictionary to map item labels to QIDs
+label_to_qid = {}
+for item in query_data:
+    label = item.get('itemLabel')
+    qid = item.get('item', '').split('/')[-1]
+    if label and qid:
+        label_to_qid[label.lower()] = qid
+
+# Read the categories.wip.txt file
+with open('/Users/pierre/development/openfoodfacts-server/taxonomies/food/ingredients.txt', 'r') as f:
+    categories_content = f.read()
+
+# Process the categories.wip.txt file
+updated_content = categories_content
+updated_count = 0
+
+for label, qid in label_to_qid.items():
+    # We need to be careful with special characters in the label
+    escaped_label = re.escape(label)
+    
+    # The label can be a value of any key.
+    # The pattern looks for a line starting with a language code, a colon, and then the label.
+    pattern = r"^[a-z]{2,3}:\s*.*" + escaped_label
+    
+    for match in re.finditer(pattern, updated_content, re.IGNORECASE | re.MULTILINE):
+        start_index = match.start()
+        
+        # Find the start of the block
+        block_start = updated_content.rfind('< en:', 0, start_index)
+        if block_start == -1:
+            continue
+            
+        # Find the end of the block
+        block_end = updated_content.find('\n\n', start_index)
+        if block_end == -1:
+            block_end = len(updated_content)
+            
+        block = updated_content[block_start:block_end]
+        
+        # Check if the block has a #wikidata:en: line
```

Comment:
Would it make sense to also have an `else if 'wikidata:en:' not in block:`? Or maybe even wrap this in that check like

```python
if 'wikidata:en:' not in block:
    if '#wikidata:en:' in block:
        …
    else:
        …
    (possibly the updated_content, _count, print, and break here)
```

There is bound to be a bunch of potential candidates that don’t have a commented-out `wikidata:en` line. But maybe it’s too cumbersome to figure out where to insert it?